package.json

```diff
@@ -29,6 +29,7 @@
     "eslint-config-next": "15.0.3",
     "focus-visible": "^5.2.0",
     "lodash.debounce": "^4.0.8",
+    "mustache": "^4.2.0",
```

Comment:
since this is also used in cognition, we can add it to the next parent image. I have some task to update the packages at some point, so maybe some note somewhere so i don't forget to add this.

Reply:
removed since was intially planned for the UI but not needed anymore